tsconfig.json

```diff
@@ -1,27 +1,14 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "noEmit": true,
     "noImplicitAny": false,
+    "skipLibCheck": true,
+    "strict": true,
     "strictNullChecks": false,
     "target": "es5",
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "declaration": true,
-    "lib": ["es7", "dom"],
-    "strict": true,
-    "skipLibCheck": true,
-    "types": ["node"],
-    "typeRoots": ["node_modules/@types"],
-    "baseUrl": ".",
-    "outDir": "lib",
-    "paths": {
-      "@/*": [
-        "src/*"
-      ]
-    }
+    "types": ["node"]
```

Comment:
Just to give some context to these changes, I made an update so that Vite/Rollup handles bundling the types, which eliminates the need for many of these properties. The `tsconfig.json` is obviously still needed for type checking, but I removed properties that dealt specifically with how Typescript would emit JavaScript and type declaration files. There are still some that I included such as `target` because Rollup takes the property into account when it's defined.